realtime-trade-monitor/README.md

```diff
@@ -67,43 +67,109 @@ Browse to localhost:9000 to see the dashboard.
 
 1. Create Enterprise Hazelcast cluster in https://cloud.hazelcast.com/
 
-2. Open client configuration window on the cluster details page and grab cluster id and discovery token. 
+2. Open client configuration window on the cluster details page and grab:
+   - Cluster group name;
+   - Cluster discovery token. 
+   
+   If during the cluster creation you've turned on TLS encryption, you should also:
+     - grab Keystore and truststore password;
+     - download and extract Keystore file.
 
 3. Modify Hazelcast client configs `trade-queries/src/main/resources/hazelcast-client.yaml` and
    `webapp/src/main/resources/hazelcast-client.yaml` 
-   by putting there cluster id and discovery token from previous step. Minimal working config example:
+
+If you're not using TLS encryption for your cluster, use the following minimal working configuration:
 ```
 hazelcast-client:
-  cluster-name: <CLUSTER_NAME>
+  cluster-name: <CLUSTER_GROUP_NAME>
   instance-name: query-client
   properties:
     hazelcast.client.cloud.url: "https://uat.hazelcast.cloud" #Optional, if env is not default
```

Comment:
Stay consistent with the environemnts. The first step expected the cluster to be created on PROD.